cabal.project

```diff
@@ -31,6 +31,10 @@ package bitvec
   -- Workaround for windows cross-compilation
   flags: -simd
 
+constraints:
+  -- io-classes-mtl-0.1.2.0 is not buildable
+  io-classes-mtl < 0.1.2.0
```

Comment:
Strange, what was the compiler error?

Reply:
https://github.com/IntersectMBO/cardano-cli/actions/runs/9254721425/job/25457077489?pr=777#step:11:973